policyengine_us/parameters/gov/states/ny/tax/income/deductions/itemized/reduction/amount/denominator.yaml

```diff
@@ -0,0 +1,14 @@
+description: New York sets this value as the denominator in calculating the percentage reduction of itemized deductions for high-income taxpayers.
+values:
+  2021-01-01: 50_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: New York itemized deduction reduction calculation denominator
+  reference:
+    - title: SECTION 615 New York itemized deduction of a resident individual (f)(2)
+      href: https://www.nysenate.gov/legislation/laws/TAX/615
+    - title: 2021 NY Form IT-196 Instructions Line 46
```

Comment:
Just for completeness, could you add 2022, 2023 tax forms as well to all yaml files